en/canary-deployment-tidb-operator.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Canary Deployment of TiDB Operator
+summary: Learn how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+---
+
+# Canary Deployment of TiDB Operator
+
+This document describes how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+
+## Scenarios
+
+- Canary upgrade TiDB Operator.
+- Deploy multiple TiDB Operator to manage different TiDB clusters.
+
+    > **Note:**
+    >
+    > - Currently, you can only deploy multiple `tidb-controller-manager`s and `tidb-scheduler`s.
+    > - Only one Admission Webhook is required.
+    > - If you have deployed multiple TiDB Operator and only some of them enable [Advanced StatefulSet](advanced-statefulset.md), the same TidbCluster Custom Resource (CR) cannot be switched among these TiDB Operator.
```

Comment:
`TiDB Operator` here looks like a countable noun but it cannot be suffixed with `s` to indicate its plural form, which to me is a bit strange.